### PR TITLE
FDM-143

### PR DESCRIPTION
**Pull request by linavo**
Changes to admin page

### CODE REVIEW

File: app/components/admin/adminModal.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div style="position:absolute; width:100%; top:110px; height:700px" class="sdm-admin-modal">
     <sdm-admin-modal>
-        <div class="admin-title">Admin Panel</div>
+        <div class="admin-title">Group Admin<!-- Admin Panel --></div>
```
Comment (rentzso) [inline app/components/admin/adminModal.html]: probably this title is a little misleading and it should be removed completely

Comment (linavo) [inline app/components/admin/adminModal.html]: Just followed comp that Elisa gave me.

Comment (linavo) [inline app/components/admin/adminModal.html]: Totally good point though, let me change that back

File: app/components/admin/adminModal.html [hunk 2]
```diff
@@ -66,6 +69,8 @@
                                 <div class="sdm-input-label">Add User to Group:<b> {{sdmAMController.selectedGroup.name}}</b></div>
                                 <input class="form-control typeahead" type="text" ng-model="sdmAMController.selectedUID" placeholder="{{sdmAMController.permissionPlaceholder}}" name="newPermission" ng-disabled="!sdmAMController.selectedGroup">
                             </div>
+                            <div class="sdm-input-cell sdm-small2"></div>
```
Comment (rentzso) [inline app/components/admin/adminModal.html]: these two columns exists only in the "table" header.
They should be added also in the "table" body

File: app/components/admin/adminModal.less [hunk 3]
```diff
@@ -169,6 +192,9 @@ div.sdm-edit-groups {
                     &.select {
                         padding-left: 0px;
                     }
+                    .fa-times-circle-o {
+                        margin-left: 175px;
```
Comment (rentzso) [inline app/components/admin/adminModal.less]: the remove icon was centered and aligned with the Remove label in the header.
If you follow my other comment on the table layout, this margin shouldn't be needed.

**Pull request merged**